components/Tabs/examples/supplemental/TabBarViewControllerExampleSupplemental.h

```diff
@@ -38,7 +41,15 @@
 @end
 
 @interface TBVCSampleViewController : UIViewController
-+ (nonnull instancetype)sampleWithTitle:(nonnull NSString *)title color:(nonnull UIColor *)color;
+
 @property(nonatomic, nullable) MDCSemanticColorScheme *colorScheme;
 @property(nonatomic, nullable) MDCTypographyScheme *typographyScheme;
+
++ (nonnull instancetype)sampleWithTitle:(nonnull NSString *)title color:(nonnull UIColor *)color;
+
+- (void)addMDCButtonWithFrame:(CGRect)frame
+                 buttonScheme:(id<MDCButtonScheming>)buttonScheme
+                        title:(NSString *)title
+                  actionBlock:(MDCButtonActionBlock)actionBlock;
```

Comment:
Please add nullability specifiers for these arguments.

Reply:
Addressed in a46864d6635c5903a535e6436aed7fb783f084b3.